### DISCUSSION TITLE
Cosmic desktop remote mouse doesn't work.

### DISCUSSION THREAD

**Discussion by zachvlat** [andyholmes/valent#764] [Feature Request]
Pretty much what the title says. Notifications and commands are working though.

**Comment by rohmishra** [comment]
that would require cosmic specific bindings or extension to work as it's a completely different DE.

**Comment by orowith2os** [comment]
> that would require cosmic specific bindings or extension to work as it's a completely different DE.

Not necessarily correct. Valent uses the xdg-desktop-portal Remote Desktop portal for remote input. However, this issue can't be solved without more information, such as logs.

**Comment by andyholmes** [comment]
Do you happen to know if kdeconnect-kde works on Cosmic? I see their Wayland implementation also uses `org.freedesktop.portal.RemoteDesktop`, but they make direct calls instead of using libportal.

**Comment by andyholmes** [comment]
Moving this to discussions, as it seems like a feature request.
